### DISCUSSION TITLE
How to set two spray exits with different liquid fuels?

### DISCUSSION THREAD

**Discussion by Rachel0096** [AMReX-Combustion/PeleC#835] [Q&A]
I apologize for any inconvenience this may cause. I would like to set up a configuration as shown in Figure 1, which has two spray exits that simultaneously different liquid fuels. I have seen in the PELEMP user manual that it supports multiple liquids, but I am not sure if my configuration can be achieved. Could you please provide guidance on this?
![7fd660cb7b1a7663e60d1e157a042bd](https://github.com/user-attachments/assets/95abf033-4a12-4f10-990d-0b231af97f23)
![5995796eb1d3c448bbf26d858bdafc5](https://github.com/user-attachments/assets/8a0dc7d4-5aea-47ac-ba9e-10f0b3ee4558)

**Comment by baperry2** [comment]
This configuration should be able to be simulated in PeleC just fine. For an example with multiple injections in multiple directions, see `Exec/RegTests/SprayMultijet`. You could easily modify the function that creates the jets to change the composition and location of the jets being created to meet your needs: https://github.com/AMReX-Combustion/PeleC/blob/3927cfcb7e5536df54daba563190cbbf84f929ae/Exec/RegTests/Spray-Multijet/SprayParticlesInitInsert.cpp#L89
